src/unstructured_client/models/errors/unstructuredclienterror.py

```diff
@@ -0,0 +1,26 @@
+"""Code generated by Speakeasy (https://speakeasy.com). DO NOT EDIT."""
+
+import httpx
+from typing import Optional
+
+
+class UnstructuredClientError(Exception):
+    """The base class for all HTTP error responses."""
+
+    message: str
+    status_code: int
+    body: str
```

Comment:
fyi we raise body back to plugins as `httpexception(detail=e.body)`

Reply:
and i think the message here could be something like
`"API error occurred: Status 402 Content-Type application/json. Body: {\"detail\": \"api key Not found etc\"}"`

Reply:
Yeah, I would have preferred it if they didn't concatenate everything like that. At least `body` will still be the actual json so we can pull out the server response.